src/amo/components/SearchResult/styles.scss

```diff
@@ -145,6 +145,8 @@
 }
 
 .SearchResult-author {
+  @include margin-end(10px);
```

Comment:
Were you trying to match something else that is already 10px? I ask because our UI guidelines are to only use multiples of 2. The closest would be 12px. However, there are some old styles that don't follow this.